force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +351,19 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+           Integer position ;
         // Loop through the list of words
             // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+             for (integer i = 0; i < words.size(); i++) {
+                if (words.get(i) == wordToFind ) {
+                    position = i;
+                    break;
+                } else if (words.contains(wordToFind)  == false) {
```

Comment:
It is good you found the List method that determines if there is a specific value in the list, but it is not great to do this check within each loop because it always returns the same for the entire list. I would move this before the list and then You would not need to do the loop at all if words does not contain wordToFind. Alternatively, since you found this method, can you find another method on the List class that will do what this question is asking in 1 step?